crates/transaction-emitter-lib/src/emitter/test_proofs_for_localnet_txn_emitter.txt

```diff
@@ -0,0 +1 @@
+todo: populate this
```

Comment:
so if this file has n lines, then we can create no more than n accounts for txn emitter,  which is a reason we want proof simulator?

Reply:
for local, it helps.  for devnet and testnet idk